frontend/packages/console-plugin-sdk/src/store.ts

```diff
@@ -197,12 +198,16 @@ export class PluginStore {
     }
   }
 
-  private isDynamicPluginLoaded(pluginName: string) {
-    return Array.from(this.loadedDynamicPlugins.values()).some(
+  private getLoadedDynamicPlugin(pluginName: string) {
+    return Array.from(this.loadedDynamicPlugins.values()).find(
       (plugin) => plugin.manifest.name === pluginName,
     );
   }
 
+  private isDynamicPluginLoaded(pluginName: string) {
+    return this.getLoadedDynamicPlugin(pluginName) !== undefined;
```

Comment:
nit: Use double negation `!!this.getLoadedDynamicPlugin(pluginName)` to make this more concise.